src/components/AssetHeader/AssetActions.tsx

```diff
@@ -109,62 +113,84 @@ export const AssetActions: React.FC<AssetActionProps> = ({
     navigate(`/trade/${assetId}`)
   }, [assetId, navigate])
 
+  const handleMoreClick = useCallback(() => {
+    setIsMoreActionsOpen(true)
+  }, [])
+
+  const handleMoreClose = useCallback(() => {
+    setIsMoreActionsOpen(false)
+  }, [])
```

Comment:
nitpick: no need to tackle here, but feels like a `useToggle()` would've been a good contender?